src/components/CarouselClassTalentsSelect/index.tsx

```diff
@@ -15,7 +15,7 @@ export default function SelectTalents(props: Props) {
     .filter((talent) => !props.equippedTalents.includes(talent.name))
     .map((talent) => ({
       name: talent.name,
-      renderNode: () => <CardTalent title={talent.name} description={talent.rawDescription} />,
+      renderNode: () => <CardTalent talent={talent} key={talent.name} />,
```

Comment:
/nit: I don't think this needs a `key`, unless Talents are rendered differently - these get wrapped in an element in the Carousel which has the key. Ignore this if I'm wrong  :\